.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  extends: [
+    '@elastic/eslint-config-kibana',
+    '@elastic/eslint-config-kibana/jest'
+  ],
+
+  settings: {
+    'import/resolver': {
+      '@elastic/eslint-import-resolver-kibana': {
+        rootPackageName: 'kibana',
+        kibanaPath: '.'
+      }
+    }
+  },
+
+  overrides: [
+    {
+      files: [
+        'src/server/**/__tests__/**/*.js',
```

Comment:
Until this pr `__tests__` has always meant mocha tests, while `xyz.test.js` has been used to identify jest tests. Is it too much to ask that we rename the test files to match the expected pattern rather than break from the convention?